Tasks/MavenAuthenticateV0/task.json

```diff
@@ -0,0 +1,68 @@
+{
+    "id": "D4B964F9-EA90-41BB-9526-29589628AD90",
+    "name": "MavenAuthenticate",
+    "friendlyName": "Maven Authenticate",
+    "description": "Provides credentials for Azure Artifacts feeds and service connections for maven repositories",
+    "helpUrl": "https://go.microsoft.com/fwlink/?linkid=2099968",
+    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=2099968)",
+    "category": "Package",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 0
+    },
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "minimumAgentVersion": "2.115.0",
+    "instanceNameFormat": "Maven Authenticate",
+    "inputs": [
+        {
+            "name": "feeds",
+            "type": "pickList",
+            "label": "Feeds",
+            "defaultValue": "",
+            "helpMarkdown": "Select one or multiple feeds to authenticate.",
+            "properties": {
+                "EditableOptions": "true",
+                "MultiSelectFlatList": "true"
+            }
+        },
+        {
+            "name": "serviceEndpoints",
+            "label": "Credentials for repositories outside this organization/collection",
+            "helpMarkDown": "Credentials to use for external repositories located in the project's pom.xml.",
+            "type": "connectedService:externalmavenrepository",
+            "properties": {
+                "MultiSelectFlatList": "true"
+            }
+        }
+    ],
+    "dataSourceBindings": [
+        {
+            "target": "feeds",
+            "endpointId": "tfs:feed",
+            "endpointUrl": "{{endpoint.url}}/_apis/packaging/feeds",
+            "resultSelector": "jsonpath:$.value[*]",
+            "resultTemplate": "{ \"Value\" : \"{{{name}}}\", \"DisplayValue\" : \"{{{name}}}\" }"
```

Comment:
I think we need to pass project name as well if it is project scoped feed. Otherwise the url will only contain feed name which will throw FeedNotFoundException later. Please make sure that it is covered.

Reply:
We only need the feed id in this case, the url is set by the user in the pom.xml in the users repo.